.github/workflows/pkg-tests.yml

```
@@ -77,3 +87,68 @@ jobs:
           -v $PWD:/home/workspace
           ${{ needs.extract.outputs.container }}
           /bin/sh -c "PKG_NAME=${{ matrix.package }} /home/workspace/pkg-tests.sh -A"
+
+      - name: "Get job status"
+        id: job-status
+        if: always()
+        run: |
+          PKG="${{ matrix.package }}"
+          STATUS="${{ job.status }}"
+          echo ${STATUS} > "${PKG}.txt"
+
+      - name: "Upload job status"
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: ${{ matrix.package }}
+          path: "${{ matrix.package }}.txt"
+
+  report:
+    name: "Report"
+    needs: individual-package
+    if: always()
```

Probably, at least once we have finished testing, we only want to generate reports when we have run this workflow with *all* packages.
```suggestion
    if: ${{ always() && github.event.inputs.packages == '' }}
```

---

We could also save those in a different location, for example the suggested `manual` folder that could contain all reports created by a workflow dispatch.